test-suites/impltests/powershell.go

```diff
@@ -0,0 +1,43 @@
+package impltests
+
+import (
+	"fmt"
+	test_executor "github.com/Netflix/bettertls/test-suites/test-executor"
+)
+
+// Compatibility notes:
+// This runner requires Administrator privileges so that it can add/remove root CA's.
+// You must also have "test-suites/scripts/" in your PATH.
+
+type PowerShellRunner struct {
+	version string
+}
+
+func (c *PowerShellRunner) Name() string {
+	return "powershell"
+}
+
+func (c *PowerShellRunner) Initialize() error {
+	var err error
+	c.version, err = execAndCapture("powershell", "$PSVersionTable.PSEdition + \" \" + $PSVersionTable.PSVersion")
```

Comment:
I'm not sure if this already includes it, but are you able to capture the windows version (e.g. build number) here as well? My hunch is that under the hood powershell will be using a Windows-provided certificate verification stack rather than something necessarily powershell specific, so having the Windows version would probably be useful.

Reply:
Yeah that probably would be helpful; I'll update this PR shortly.

Reply:
PR updated.